docs/flexibility/trigger-updates.md

```diff
@@ -0,0 +1,58 @@
+# Trigger updates
+
+Clients can issue simultaneous queries for a table. The [`table_open_cache`](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_table_open_cache) system variable returns the number of open tables for all threads. Raising this value increases the number of file descriptors. The [`table_open_cache_instances`](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_table_open_cache_instances) shows the number of open tables cache instances.
```

Comment:
Hmm... I think description of table_open_cache variable meaning in the upstream documentation at https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_table_open_cache is not quite correct. So let us not repeat it after them!

The problem is that table_open_cache system variable sets the soft limit on maximum open tables in the cache. So a) this limit can be exceeded temporarily, b) even if there are no active threads using any table at the moment there can be open tables in this cache c) if the value is set high enough it is quite possible that there will be less open tables when specified by this limit. The real number of currently open tables is visible in Open_tables status variable...

Actually, upstream has a bit more clear description of how this variable and the whole table cache works at https://dev.mysql.com/doc/refman/8.0/en/table-cache.html.

So perhaps we can invent some better description for this as well?

Something like:

"Clients can issue simultaneous queries for a table. To avoid scalability problems each thread handling client query gets its own open table instance. Server uses special cache, called Table Cache, which contains open table instances to not pay penalty which might be caused by opening and closing tables for each statement. The table_open_cache system variable sets soft limit on the size of this cache. It can be temporary exceeded if currently executing queries require more open tables than specified by this limit. However, once such queries will complete server will close unused open table instances from this cache using LRU algorithm."

?

Reply:
Updated.